### PR TITLE
Add figure to out-of-sample pred proba via cv tutorial

### PR DESCRIPTION
**Pull request by weijinglok**
This PR adds a supporting diagram to the tutorial for computing out-of-sample predicted probabilities with cross-validation. 

A mockup is available [here](https://weijinglok.github.io/cleanlab-docs/add-cross-val-im/tutorials/pred_probs_cross_val/pred_probs_cross_val.html).

**BEFORE APPROVAL & MERGING**, do note that the image is 100KB+, hence do let me know if this should be hosted else where.

### CODE REVIEW

**Comment by jwmueller** [comment]
unrelated to this PR but can you shorten:
"What are out-of-sample predicted probabilities?" 
-> 
"Out-of-sample predicted probabilities?"

I think the former is too long and is causing a weird spacing issue below it.

File: docs/source/tutorials/pred_probs_cross_val/pred_probs_cross_val.rst [hunk 1]
```diff
@@ -21,7 +21,11 @@ When using ``cleanlab``, we will typically want to find label issues in all labe
 What is K-fold cross-validation?
 --------------------------------
 
-K-fold cross-validation partitions the entire dataset into *K* disjoint subsets of data called **folds**. *K* independent copies of our model are trained, where for each model copy, one fold of the data is held out from its training (the data in this fold may be viewed as a **validation set** for this copy of the model). Each copy of the model has a different validation set for which we can obtain out-of-sample predicted probabilities from this copy of the model. Since each datapoint is held-out from one copy of the model, this process allows us to get out-of-sample predictions for every datapoint! We, in particular, recommend applying stratified cross-validation, which tries to ensure the proportions of data from each class match across different folds.
+.. image:: pred_probs_cross_val.png
```
Comment (jwmueller) [inline docs/source/tutorials/pred_probs_cross_val/pred_probs_cross_val.rst:24]: Good call on the image size. Maybe let's move it here?

https://github.com/cleanlab/assets

Comment (weijinglok) [reply docs/source/tutorials/pred_probs_cross_val/pred_probs_cross_val.rst]: moved image to here: https://github.com/cleanlab/assets/blob/master/cleanlab/pred_probs_cross_val.png

File: docs/source/tutorials/pred_probs_cross_val.rst [hunk 2]
```diff
@@ -21,7 +21,11 @@ When using ``cleanlab``, we will typically want to find label issues in all labe
 What is K-fold cross-validation?
 --------------------------------
 
-K-fold cross-validation partitions the entire dataset into *K* disjoint subsets of data called **folds**. *K* independent copies of our model are trained, where for each model copy, one fold of the data is held out from its training (the data in this fold may be viewed as a **validation set** for this copy of the model). Each copy of the model has a different validation set for which we can obtain out-of-sample predicted probabilities from this copy of the model. Since each datapoint is held-out from one copy of the model, this process allows us to get out-of-sample predictions for every datapoint! We, in particular, recommend applying stratified cross-validation, which tries to ensure the proportions of data from each class match across different folds.
+.. image:: https://raw.githubusercontent.com/cleanlab/assets/master/cleanlab/pred_probs_cross_val.png
+    :alt: Computing Out-of-Sample Predicted Probabilities from K-Fold Cross-Validation
```
Comment (jwmueller) [inline docs/source/tutorials/pred_probs_cross_val.rst:25]: just curious what is this for?

**Review by jwmueller — APPROVED** [review]
LGTM!

**Pull request merged**